packages/terra-functional-testing/lib/docker/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
+  hub:
+   image: selenium/hub:3.141.59-zirconium
+   ports:
+     - "4444:4444"
+
+  chrome:
+    image: selenium/node-chrome:3.141.59-zirconium
+    volumes:
+      - /dev/shm:/dev/shm
+    environment:
+      HUB_HOST: hub
+      HUB_PORT: 4444
+    deploy:
+        replicas: 1
+    entrypoint: bash -c 'SE_OPTS="-host $$HOSTNAME" /opt/bin/entry_point.sh'
+
+  firefox:
+    image: selenium/node-firefox:3.141.59-zirconium
+    volumes:
+      - /dev/shm:/dev/shm
+    environment:
+      HUB_HOST: hub
+      HUB_PORT: 4444
+    deploy:
+        replicas: 1
+    entrypoint: bash -c 'SE_OPTS="-host $$HOSTNAME" /opt/bin/entry_point.sh'
```

Comment:
If you want to be cheeky,  you could use anchors to reduce your docker compose file.
```yml
  x-selenium-browser: &selenium-browser
    volumes:
      - /dev/shm:/dev/shm
    environment:
      HUB_HOST: hub
      HUB_PORT: 4444
    deploy:
        replicas: 1
    entrypoint: bash -c 'SE_OPTS="-host $$HOSTNAME" /opt/bin/entry_point.sh'

  chrome:
    << *selenium-browser
    image: selenium/node-chrome:3.141.59-zirconium

  firefox:
    << *selenium-browser
    image: selenium/node-firefox:3.141.59-zirconium
```
https://medium.com/@kinghuang/docker-compose-anchors-aliases-extensions-a1e4105d70bd

Reply:
This is a cool. I'd like to stay closely with the recommended compose file if we can:

https://github.com/SeleniumHQ/docker-selenium#version-3-with-swarm-support